packages/manager/cypress/support/ui/autocomplete-popper.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Autocomplete Popper UI element.
+ *
+ * Useful for validating content, filling out forms, etc. that appear within
+ * a autocomplete popper.
+ */
+export const autocompletePopper = {
+  /**
+   * Finds a autocomplete popper that has the given title.
+   */
+  findByTitle: (title: string): Cypress.Chainable => {
+    return cy
+      .findByText(`${title}`)
+      .closest('[data-qa-autocomplete-popper="true"]');
+  },
```

Comment:
```suggestion
  /**
   * Finds an autocomplete popper item that has the given title.
   */
  findItemByTitle: (title: string): Cypress.Chainable => {
    return cy
      .document()
      .its('body')
      .find('[data-qa-autocomplete-popper]')
      .findByText(title);
  },
```

Quick suggestion for the new UI helper! Aside from the function name and docs tweaks, selecting the popper from anywhere in the document body will allow us to use this utility from inside `within(() => { ... })` callbacks which will be more convenient in certain situations, but will still work the way you're using it now, too.

I think in the future there is potential to use this to interact with other drop-downs and autocomplete fields as well (especially the Linode region select) -- I'm expecting this to be a pretty big time saver. Nice work!